README.md

```diff
@@ -485,6 +485,28 @@ statusCode, body, err := fasthttp.Get(nil, "http://google.com/")
 uintBuf := fasthttp.AppendUint(nil, 1234)
 ```
 
+* String and `[]byte` buffer may converts wihtout memory allocation
+```go
+func b2s(b []byte) string {
+    return *(*string)(unsafe.Pointer(&b))
+}
+
+func s2b(s string) (b []byte) {
+    bh := (*reflect.SliceHeader)(unsafe.Pointer(&b))
+    sh := (*reflect.StringHeader)(unsafe.Pointer(&s))
+    bh.Data = sh.Data
+    bh.Cap = sh.Len
+    bh.Len = sh.Len
+    return b
+}
+```
+The underlying structure of `[]byte` buffer has only one `Len` field more than String.
+So, we can construct one directly from the other.
```

Comment:
I'm not sure if it's useful to mention here.

Reply:
I think some people are wondering how it works, but if people want to know more details, they'll Google it. It's ok to delete.